more_services/index.html

```diff
@@ -1,645 +1,1096 @@
+                </a>
+              </li>
+            </ul>
+            <a class="seeAllLink" href="/more-services" target="_self">
+              <span>See all</span>
+              <svg class="h2 m_h1 black-300" height="18" width="18" fill="currentColor" viewBox="0 0 18 18"
+                xmlns="http://www.w3.org/2000/svg">
+                <path
+                  d="M9.764 14.646L16 9 9.725 3.311a1 1 0 00-1.482 1.342L11.75 8H3.002A1 1 0 103 10h8.75l-3.449 3.285c-.187.2-.301.435-.301.715a1 1 0 001 1c.306 0 .537-.151.764-.354z">
+                </path>
+              </svg>
+            </a>
+            <!-- </div> -->
+          </div>
+        </div>
+      </template>
     </div>
-    <hr />
-    <div class="grid grid-cols-3 mt-16 max-w-screen-lg mx-auto">
-      <div class="col-span-1 font-mark font-normal text-base flex-col">
-        <ul>
+
+    <nav class="nav-links-mobile" x-show="open" x-cloak x-transition:enter="transition ease-out duration-200"
+      x-transition:enter-start="-translate-y-full" x-transition:enter-end="translate-y-0"
+      x-transition:leave="transition ease-in duration-200" x-transition:leave-start="translate-y-0"
+      x-transition:leave-end="-translate-y-full">
+      <a href="#">Home</a>
+      <div class="flex flex-col">
+        <a href="#" @click="showExploreMenu = !showExploreMenu">
+          <span class="flex">
+            <span> Explore </span>
+            <svg :class="showExploreMenu ? 'rotate-90' : ''" height="28" width="28" viewBox="0 0 28 28"
+              xmlns="http://www.w3.org/2000/svg">
+              <path
+                d="M10.764 21.646L19 14l-8.275-7.689a1 1 0 00-1.482 1.342L16 14l-6.699 6.285c-.187.2-.301.435-.301.715a1 1 0 001 1c.306 0 .537-.151.764-.354z">
+              </path>
+            </svg>
+          </span>
+        </a>
+        <div id="exploreMenuMobile" x-show="showExploreMenu" x-transition:enter="transition ease-out duration-200"
+          x-transition:enter-start="-translate-y-full opacity-0" x-transition:enter-end="translate-y-0 opacity-100"
+          x-transition:leave="transition ease-in duration-200" x-transition:leave-start="translate-y-0 opacity-100"
+          x-transition:leave-end="-translate-y-full opacity-0"></div>
+      </div>
+      <a href="#">Join as a pro</a>
+      <a href="#">Membership</a>
+      <a href="#">Sign up</a>
+      <a href="#">Login</a>
+    </nav>
+
+    <div id="exploreMenuDesktop" x-show="showExploreMenu" x-cloak @click.outside="showExploreMenu = false"
+      x-transition:enter="transition ease-out duration-200" x-transition:enter-start="-translate-y-full opacity-0"
+      x-transition:enter-end="translate-y-0 opacity-100" x-transition:leave="transition ease-in duration-200"
+      x-transition:leave-start="translate-y-0 opacity-100" x-transition:leave-end="-translate-y-full opacity-0"></div>
+  </div>
+  <hr>
+  <div class="more-services-page pt-16 pb-32">
+    <div class="grid grid-cols-3  max-w-screen-lg mx-auto">
```

Comment:
we can set this grid column template to 1 on mobile devices & to 3 when the screen is large enough.
To do so, we can use this `grid-cols-1 md:grid-cols-3`
This means that until 'md' (768px) the grid template would be configured to take 1 col/row, then after 'md', it'll
take 3 cols/row